static/app/views/replays/deadRageClick/exampleReplaysList.tsx

```diff
@@ -53,13 +55,13 @@ export default function ExampleReplaysList({
             'started_at',
             'urls',
           ],
-          projects: [],
+          projects: [projectId],
```

Comment:
idk how i feel about this part now that i see it.

in the cards i think it makes sense to have project as well, because in the example we can see `a.` listed twice in the left card:

here's a gloriously bad mockup with blue squares representing project name/icon
<img width="1151" alt="272408649-f7ec4815-dacc-48fc-be6a-d7963699d71c-1" src="https://github.com/getsentry/sentry/assets/187460/ee75a6a8-3b52-4175-9094-b6882515fd28">


but, idk if i like seeing `a.` twice. it feels like these type of generic selectors will take up the top rows at all times, at least with out data.

Reply:
added the platform icon with a tooltip with the project slug:
<img width="602" alt="SCR-20231003-owox" src="https://github.com/getsentry/sentry/assets/56095982/b2e50667-4eb3-41f6-a9c2-b38ca760f7b5">